packages/devextreme-react/src/core/widget-config.ts

```diff
@@ -8,6 +8,16 @@ function isIgnoredProp(name: string) {
   return name === 'children' || name === classNamePropName || elementPropNames.includes(name);
 }
 
+function getRefElement(value: any): HTMLElement {
```

Comment:
Can we use something else instead of any type?

Reply:
I tried something like Ref<HtmlElement> | HtmlElement | Ref<MockComponent>
There are some difficulties with creating MockComponent type and type casting

Reply:
instance getter is specified in generated react component
also we need type guards to differentiate between those types
so the code will be a little more complicated 
i'll prepare a solution and post it in channel and then we decide

Reply:
This `value` is a property value or a ref, so it may be `Ref<...> | any` which is effectively `any`.

See how the `props`  argument of `separateProps` is typed as `Record<string, any>`.

So I agree with Ivan, we may better leave types as they are...